### ISSUE TITLE
Cannot insert duplicate key row in object 'dbo.Block' with unique index 'Block_BlockHash'. 

### ISSUE THREAD

**Issue by gzepeda** [Azure-Samples/blockchain#46]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce

- Created a new Azure Blockchain Workbench
- Created an application for each of the samples provided on Github
- With a group of more than 20 users, we tested creating contracts and moving them along their different workflows. 
- At some point, the application stopped creating contracts showing the following error:

`It looks like something went wrong with creating contract 22. Contact your administrator for help resolving the problem.`

### Any log messages given by the failure
Checked application insights and I am seeing thousands of occurrences of the following error:

`Cannot insert duplicate key row in object 'dbo.Block' with unique index 'Block_BlockHash'. The duplicate key value is (0x4a07363c6902893bb73e61351a736a6ba67bc1766cd870d03bd381566049d448). The statement has been terminated.`

`Dapper.SqlMapper+<ExecuteImplAsync>d__37.MoveNext`

### Expected/desired behavior
We should be able to create contracts and no errors should be showing up on Application Insights.

### OS and Version?
Azure Blockchain Workbench

### Versions
September 2018 (1.3)

### Mention any other details that might be useful
I disabled all the apps but the errors are still showing up on Application Insights.

**Comment by curtisho1** [comment]
I am facing the same problem and I believe this is the bug we encounter. we need to delete all resource group and redeploy again 

https://techcommunity.microsoft.com/t5/Blockchain/Azure-Blockchain-Workbench-Hotfix-version-1-3-2-available/m-p/266027#M373

**Comment by gzepeda** [comment]
Quick update on this: I changed the index called Block_BlockHash on the Block table to non-unique. I don't think a hash has to be unique... correct me if I am wrong. 

The exceptions from Application insights are gone but the system is currently generating more and more records on the Block table and nobody is doing anything on the system. It appears messages on the Service Bus are being recycled and ran over and over again generating this traffic. 

I will shut down the worker VMs to stop generating traffic awaiting for some feedback on this and proceed to create a new Azure Blockchain Workbench to continue testing, since at this point I cannot create contracts on any App.

**Comment by gzepeda** [comment]
Update: After recreating the Workbench with Version 1.3.2, at first everything worked normally. 

I deployed one of the sample application "Digital Locker" and on the second step of the workflow a similar error is happening. 

Message: This is taking longer than expected, Contact your administrator... 

![image](https://user-images.githubusercontent.com/4001077/46553821-beb7c200-c89b-11e8-826f-a31814503c01.png)

Looking for errors in App Insights, the same error that version 1.3 had is reappearing. 

```
Cannot insert duplicate key row in object 'dbo.Block' with unique index 'Block_BlockHash'. The duplicate key value is (0xe74a9c372898b1c4fdcb2eb354fcff22148c30aa5b97b85124d6df132abc5517). The statement has been terminated.
Problem Id:System.Data.SqlClient.SqlException at Dapper.SqlMapper+<ExecuteImplAsync>d__37.MoveNext
```

**Comment by Guitrum** [comment]
Any updates on this, I am experiencing the same thing

**Comment by brendalee** [comment]
Apologies for the delayed response. We had some issues with the 1.3 release. Please upgrade to 1.4, thanks!

**Comment by mmercuri** [comment]
Echoing Brenda, apologies for the inconvience. This is addressed in 1.4.
